templates/ai/worker/do/openai/schema.ts

```diff
@@ -119,7 +124,7 @@ const SimpleArrowShape = z.object({
 	x2: z.number(),
 	y2: z.number(),
 	color: SimpleColor.optional(),
-	text: SimpleLabel.optional(),
+	text: z.string().optional(),
```

Comment:
i'm considering adding rich text to arrows btw: https://github.com/tldraw/tldraw/pull/6325
maybe this should be SimpleLabel just in case?